recipe/meta.yaml

```diff
@@ -20,38 +17,47 @@ source:
     - gvconfig_libdir.patch  # [unix]
 
 build:
-  number: 7
+  number: 8
   detect_binary_files_with_prefix: True  # [unix]
 
 requirements:
   build:
-    - cairo 1.14.*             # [linux]
-    - expat 2.2.*                   # [linux]
-    - libpng >=1.6.32,<1.6.35  # [unix]
-    - libtiff >=4.0.8,<4.0.10   # [unix]
-    - libtool                  # [linux]
-    - freetype 2.8.1             # [unix]
-    - jpeg 9*                  # [unix]
-    - pango 1.40.*             # [linux]
+    - {{ compiler('cxx') }}    # [not osx]
```

Comment:
Nothing is compiled on Windows. The binaries provided by Graphviz are just repackaged as a conda package, so this makes no sense.

Reply:
I know, but removing it caused it to fail on win (see [here](https://ci.appveyor.com/project/conda-forge/graphviz-feedstock/build/1.0.42)). I don't know why either. But it also does not really hurt, because we might want to compile in the future.